thorlcr/activities/piperead/thprslave.cpp

```diff
@@ -455,7 +463,6 @@ class CPipeThroughSlaveActivity : public CPipeSlaveBase
             throw wrexc.getClear();
         if (retcode!=0 && !(flags & TPFnofail))
             throw MakeActivityException(this, TE_PipeReturnedFailure, "Process returned %d", retcode);
-        PARENT::stop();
```

Comment:
Not really related, but noticed during change it was already called a few lines before (on line 460)